js/src/common/components/ModalManager.tsx

```diff
@@ -24,16 +24,21 @@ export default class ModalManager extends Component<IModalManagerAttrs> {
 
   view(vnode: Mithril.VnodeDOM<IModalManagerAttrs, this>): Mithril.Children {
     const modal = this.attrs.state.modal;
+    const Tag = modal?.componentClass;
 
     return (
       <div className="ModalManager modal fade">
-        {!!modal &&
-          modal.componentClass.component({
-            ...modal.attrs,
-            animateShow: this.animateShow.bind(this),
-            animateHide: this.animateHide.bind(this),
-            state: this.attrs.state,
-          })}
+        {!!Tag && (
+          <div>
```

Comment:
What's the need for this extra `<div>`?

Reply:
Accidentially left that in, I thought I had removed that.